app/helpers/grouping_helper.rb

```diff
@@ -18,4 +18,12 @@ def area_tag(area, *classes)
                 :'data-y' => y,
                 style: "transform: translate(#{x}px, #{y}px)")
   end
+
+  class ActionView::Helpers::FormBuilder
```

Comment:
I'd create a new file under `helpers` called `GroupingFormBuilder` that inherits from `ActionView::Helpers::FormBuilder`. That way in your form you can do
```erb
<%= form_for ..., builder: GroupingFormBuilder do |f| %>
```

More info [here](https://mixandgo.com/blog/understanding-and-extending-rails-form-helpers)

I personally try to stay away from multiclass files whenever I can.